src/model/message.ts

```diff
@@ -1,32 +1,23 @@
-import { DMChannel, GroupDMChannel, Message, TextChannel } from 'discord.js'
+import { DMChannel, GroupDMChannel, Message, Structures, TextChannel } from 'discord.js'
 import { CommandoClient } from '.'
 import { Command } from './command'
 
 /**
- *  Subclass of the Discord.js Message class.
- *
- * @todo
+ * Subclass of the Discord.js Message class.
  */
 export class CommandoMessage extends Message {
-  /**
-   * The base client object.
-   */
-  public client: CommandoClient
-
   /**
    * The command that was ran.
    */
-  public command: Command
+  public command: Command | undefined = undefined
```

Comment:
This doesn't look like its ever getting set.

Reply:
https://github.com/Nightwatch/commando-lite/pull/1/files#diff-d35742aea0e51445752b3d44b8c6d1e4R287